jvm-packages/xgboost4j/src/test/scala/ml/dmlc/xgboost4j/scala/DMatrixSuite.scala

```diff
@@ -167,4 +167,52 @@ class DMatrixSuite extends FunSuite {
     assert(dmat0.rowNum === 10)
     assert(dmat0.getLabel.length === 10)
   }
+
+    test("create DMatrix from DenseMatrix") {
```

Comment:
indention here is incorrect,

Reply:
@CodingCat Can we set up lint / style check for Java and Scala code? Currently, Python and C++ code are automatically checked for style.

Reply:
@hcho3 There is linting that this PR is failing. The main thing is that you can pass the lint filter but not meet the style guide, e.g. my other PR where I used Scalafmt.